platform/android/export/export_plugin.cpp

```diff
@@ -836,6 +836,12 @@ void EditorExportPlatformAndroid::_get_permissions(const Ref<EditorExportPreset>
 				r_permissions.push_back("com.oculus.permission.HAND_TRACKING");
 			}
 		}
+		bool use_anchor_api = p_preset->get("xr_features/use_anchor_api");
```

Comment:
@konczg Can you move these configs to the Godot OpenXR loader plugin as mentioned in https://github.com/godotengine/godot/pull/68259/files#r1265943701

Reply:
Thanks for the update! Let me know if you need assistance with updating the Godot OpenXR loader plugin with the anchor configs.